src/modules/auto-flickity.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @author Deux Huit Huit
+ * 
+ * Auto flickity module
+ */
+(function ($, undefined) {
+
+	'use strict';
+	
+	var win = $(window);
+	var site = $('#site');
+	var page = $('.page');
+
+	/*
+		Front-end Integration Hierarchy:
+
+		|- FLICKITY CTN : .js-auto-flickity-slider-ctn
+		|		|- CELL-CTN : .js-auto-flickity-ctn
+		|		|		|- CELL (REPEATED): .js-auto-flickity-item
+		|		|		|
+		|		|- NAV BTNS: .js-auto-flickity-nav-btn
+
+
+		Notes:
+
+		Flickity wont be activated if only one cell(o.cellSelector) is
+		detected. It will add the aborted class(o.abortedCl) on the cell-ctn(o.cellCtn)
+
+		You can have more info on the options of Flickity at
+		http://flickity.metafizzy.co/
+
+	 */
+
+	var o = {
+		sliderCtn: '.js-auto-flickity-slider-ctn',
+		cellCtn:'.js-auto-flickity-ctn',
+		cellSelector: '.js-auto-flickity-item',
+		navBtnSelector:'.js-auto-flickity-nav-btn',
+		abortedCl:'js-auto-flickity-cancelled',
```

Comment:
No. It's the class added if Flickity wont be inited because there's only one cell present.